src/components/Graph/Graph.tsx

```diff
@@ -113,43 +123,15 @@ const Graph: FC<GraphProps> = ({
     }
   }, [expandCurrentNode, path, setExpanded]);
 
+  // Recalculate graph layout if window size changes or if a node is expanded
```

Comment:
Its not a huge deal because the graph layout algorithm is running really quickly but it looks like expanding a node causes the layout algorithm to run three times

Reply:
True, this looks like it's also happening on `master` though.  I guess if the app gets really slow we can create a task to fix this.

Reply:
We have a task for that: PATHWAYS-144